build.gradle

```diff
@@ -6,19 +6,26 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:4.1.2"
+        implementation 'com.github.euphony-io:euphony:562a8fab8d'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
-
 allprojects {
     repositories {
         google()
         jcenter()
+        maven { url 'https://jitpack.io' }
     }
 }
-
 task clean(type: Delete) {
     delete rootProject.buildDir
+}
+android {
+    compileSdkVersion 19
+    buildToolsVersion '19.1.0'
+}
```

Comment:
A build error occurs at this point.
```
android {
    compileSdkVersion 19
    buildToolsVersion '19.1.0'
}
```
Checking the build.gradle.app , it appears that this code already exists and no longer needs to be written.